panoptes/parser_utils.py

```diff
@@ -1,17 +1,14 @@
-import os
```

Comment:
think this file (parser_utils.py) we can delete in general

Reply:
Deleted! Thanks for the suggestion!